internal/connectors/plugins/public/powens/client/bank_accounts.go

```diff
@@ -21,6 +21,7 @@ type BankAccount struct {
 	Error        string    `json:"error"`
 	LastUpdate   time.Time `json:"last_update"`
 
+	Balance      float64       `json:"balance"`
```

Comment:
In addition to my comment about `json.Number` the docs say this field could be decimal or nil, so I think we have to prepare for nil values